python/ray/data/read_api.py

```diff
@@ -320,6 +323,86 @@ def read_datasource(
     )
 
 
+@PublicAPI(stability="alpha")
+def read_mongo(
+    uri: str,
+    database: str,
+    collection: str,
+    pipelines: List[List[Dict]],
```

Comment:
wondering why `pipelines` is required? What if user just wants to read all content in the collection without any filtering?